common/src/main/java/org/bf2/common/OperandUtils.java

```diff
@@ -46,6 +47,12 @@ public static Map<String, String> getDefaultLabels() {
         return result;
     }
 
+    public static Map<String, String> getMasterSecretLabel() {
+        LinkedHashMap<String, String> result = new LinkedHashMap<>(1);
+        result.put(MANAGED_BY_LABEL, MASTER_SECRET_NAME);
+        return result;
+    }
```

Comment:
I see that this method is used for filtering the secrets used by the informer, but the labels are not being set on the secrets. Rather than `managed-by`, I wonder if setting something like `app.kubernetes.io/component=master-secret` would make more sense for this secret.

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

Reply:
+1

Reply:
component would be fine, or managed/created-by sync.

Reply:
Changed it to `app.kubernetes.io/component=master-secret`